examples/src/main/scala/org/apache/spark/examples/pythonconverters/HBaseConverters.scala

```diff
@@ -23,15 +23,27 @@ import org.apache.spark.api.python.Converter
 import org.apache.hadoop.hbase.client.{Put, Result}
 import org.apache.hadoop.hbase.io.ImmutableBytesWritable
 import org.apache.hadoop.hbase.util.Bytes
+import org.apache.hadoop.hbase.KeyValue.Type
+import org.apache.hadoop.hbase.CellUtil
 
 /**
- * Implementation of [[org.apache.spark.api.python.Converter]] that converts an
- * HBase Result to a String
+ * Implementation of [[org.apache.spark.api.python.Converter]] that converts all
+ * the records in an HBase Result to a String
  */
 class HBaseResultToStringConverter extends Converter[Any, String] {
   override def convert(obj: Any): String = {
+    import collection.JavaConverters._
     val result = obj.asInstanceOf[Result]
-    Bytes.toStringBinary(result.value())
+    val output = result.listCells.asScala.map(cell =>
+        "{'columnFamliy':'%s','qualifier':'%s','timestamp':'%s','type':'%s','value':'%s'}".format(
+          Bytes.toStringBinary(CellUtil.cloneFamily(cell)),
+          Bytes.toStringBinary(CellUtil.cloneQualifier(cell)),
+          cell.getTimestamp.toString,
+          Type.codeToType(cell.getTypeByte),
+          Bytes.toStringBinary(CellUtil.cloneValue(cell))
+        )
+    )   
+    output.mkString(" ")
```

Comment:
In fact, in HBase a A *{row, columnFamily:qualifier, version}* tuple exactly specifies a cell (a record) in HBase. So there are usually several records per column family in a row. 
In fact, HBase `Result` contains all the information about a HBase table and `Result.listCells` returns all the cells as a java list of `hbase.Cell` which contains all the information about a cell.